src/main/java/bob/Bob.java

```diff
@@ -0,0 +1,96 @@
+package bob;
+import exceptions.CommandNotFoundException;
+import exceptions.MissingParamsException;
+import exceptions.PositionException;
+import misc.Parser;
+import misc.Storage;
+import misc.Ui;
+import task.Tasklist;
+
+
+public class Bob {
+    Storage storage;
+    Tasklist tasklist;
+    Ui ui;
+    Parser parser;
+
+    public Bob() {
+        this.storage = new Storage("src/main/data/dataFile.txt");
+        this.tasklist = new Tasklist();
+        this.parser = new Parser();
+        this.ui = new Ui();
+    }
+
+    public void run() throws CommandNotFoundException, MissingParamsException, PositionException {
+        // init items
+        boolean exit = false;
+        String[] strArr = new String[2];
```

Comment:
Maybe `strArr` can be renamed as `inputTokens` instead so it is clear that this is a multi-valued variable.